wgengine/router/ifconfig_windows.go

```diff
@@ -332,21 +319,17 @@ func configureInterface(cfg *Config, tun *tun.NativeTun) error {
 		}
 		deduplicatedRoutes = append(deduplicatedRoutes, &routes[i])
 	}
-	log.Printf("routes: %v", routes)
 
 	var errAcc error
 	err = iface.SyncRoutes(deduplicatedRoutes)
 	if err != nil && errAcc == nil {
-		log.Printf("setroutes: %v", err)
```

Comment:
I'm fine making this all quieter, but I'd keep the log.Prints in the error paths. Those have been very useful during debugging.